.scripts/run_config.sh

```diff
@@ -52,7 +52,7 @@ if [ $target == "Default" ]; then
 
 elif [ "$target" == "BigQuery" ]; then
 
-  BIGQUERY_CREDS=${BIGQUERY_CREDS:-$CREDENTIALS}
```

Comment:
I'm on board with changing the behaviour here to point to a file rather than a blob of JSON, two comments on that:

- Let's make sure the readme is updated with it
- If we're doing this, we can just call the env var GOOGLE_APPLICATION_CREDENTIALS and remove this whole piece of logic :)

Reply:
Oh and also if we do that we should remove the bit that deletes the credentials file.